ui/src/style/pdf-viewer.less

```diff
@@ -2,11 +2,14 @@
  * Move view of PDF page inward when the drawer is open.
  * TODO(andrewhead): Find a way to animate drawer even when left sidebar is open.
  */
-#viewerContainer {
+.page {
   transition-property: "padding-right";
 
   &.drawer-open {
-    padding-right: @drawer-paper-clipping-width + @margin * 2;
+    padding-right: @drawer-paper-clipping-width + @margin * 4;
```

Comment:
I found * 4 looks slightly better

Reply:
If I have time, I might try to come up with a more verbose/mathy solution to this (instead of just @margin * 4)